scripts/clean-changelog-unreleased.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const globby = require("globby");
+
+const changelogUnreleasedDir = path.join(__dirname, "../changelog_unreleased");
+
+// Remove blog-post-intro.md
+const introFile = path.join(changelogUnreleasedDir, "blog-post-intro.md");
+fs.unlinkSync(introFile);
+
+// Remove change files
+const files = globby.sync("*/*.md", { cwd: changelogUnreleasedDir });
```

Comment:
```suggestion
const files = globby.sync(["blog-post-intro.md", "*/*.md"], {
  cwd: changelogUnreleasedDir,
});
```

Remove `blog-post-intro.md` too?

Reply:
And `globby` support `absolute` options, no need join yourself.